apps/backend/src/db/migrations/20240215141622_create_friends_table.ts

```diff
@@ -0,0 +1,49 @@
+import { type Knex } from "knex";
+
+const TABLE_NAME = {
+	USER_FOLLOWERS: "user_followers",
```

Comment:
```suggestion
	USER_FOLLOWERS: "user_friends",
```
please use it everywhere (in all files). if you want to change `friends` to `followers`, than you also need to change frontend part of the app

Reply:
I corrected the table.
But I have my doubts.
Maybe we should really rename everything to friend -> follower?
If you agree with me, I will do it right now.